shakenfist/net.py

```diff
@@ -276,11 +309,28 @@ def delete(self):
                         ipm.release(self.floating_gateway)
                         db.persist_ipmanager('floating', ipm.save())
 
+    def is_dnsmasq_running(self):
+        """Determine if dnsmasq process is running for this network"""
+        try:
+            out, _ = processutils.execute('pgrep -a dnsmasq', shell=True)
```

Comment:
So, there should be lots of dnsmasq processes (one per network, they're in a network namespace). So this wont work? That said, we also have the pid of the dnsmasq process at /srv/shakenfist/dhcp/...net...uuid.../pid so we could do a more direct "is pid running" check with psutils?

Reply:
The process is affecting the network namespace but the linux process list is unaffected. 

"Works on my cluster" (TM)

Reply:
We've just talked about this and I think we are aligned now.